File: src/main/java/cn/lnkdoc/sdk/uia/common/GrantType.java
package cn.lnkdoc.sdk.uia.common;

/**
 * @author langkye
 * @since 1.0.0.RELEASE
 */
public enum GrantType {
    /**
     * authorization_code
     */
    AUTHORIZATION_CODE("authorization_code")
    ;
    
    private final String code;

    GrantType(String code) {
        this.code = code;
    }

    public String getCode() {
        return code;
    }
}

File: src/main/kotlin/cn/lnkdoc/sdk/uia/common/GrantType.kt
package cn.lnkdoc.sdk.uia.common

/**
 * @author langkye
 * @since 1.0.0.RELEASE
 */
@Suppress("unused")
enum class GrantType(val code: String) {
    /**
     * authorization_code
     */
    AUTHORIZATION_CODE("authorization_code"),
    /**
     * refresh_token
     */
    REFRESH_TOKEN("refresh_token"),
    ;
}